config/settings.yml

```diff
@@ -1,3 +1,6 @@
+app_name: 'compliance'
+path_prefix: '/api'
+old_path_prefix: '/r/insights/platform'
```

Comment:
These defaults make sense for all environments. They are still customizable via env variables, like `SETTINGS__APP_NAME` if required.